source/user-manual/elasticsearch/elastic-tuning/troubleshooting.rst

```diff
@@ -173,7 +173,8 @@ Wazuh Kibana plugin page goes blank
 
 Sometimes, after an upgrade, the Wazuh Kibana plugin page goes blank. This is due to some issues with the cache memory of the browser.
```

Comment:
- Update the Troubleshooting section and include common problems and solutions for the Wazuh dashboard.